### DISCUSSION TITLE
I've recently started using the new version of oh-my-posh and cant figure out how to update the git thing

### DISCUSSION THREAD

**Discussion by RanvirChoudhary** [JanDeDobbeleer/oh-my-posh#2217] [Troubleshoot] [answered]
In the previous version of oh-my-posh i think paradox theme had this feature where if you had like uncommited changes or unpushed changes, the color of the "tag"? would change from green to yellow. i tried to mess around with the powershell files(i really like the new system it's much easier now) and i managed to change a few things like color and stuff but i cant figure out how to bring that git changing colors feature back. can i get some help pls! :)

**Comment by lewis-yeung** [comment, marked as answer]
Just read through the documentation and you will get how to tweak a `git` segment using `background_templates`.

- Ref-1: [Color templates](https://ohmyposh.dev/docs/configuration/colors#color-templates)
- Ref-2: [Sample config of a `git` segment](https://ohmyposh.dev/docs/segments/git#sample-configuration)

**Reply by RanvirChoudhary** [reply]
Thanks for your help!
